git-hg/diff.sh

```diff
@@ -53,6 +54,17 @@ function cloneRepos() {
 
   git clone -b master "https://github.com/AdoptOpenJDK/openjdk-${git_repo_version}.git" openjdk-git || exit 1
   hg clone "https://hg.openjdk.java.net/${hg_root_forest}/${hg_repo_version}" openjdk-hg || exit 1
+
+  if [ -n "${tag}" ]; then
+    cd openjdk-hg
+    hg update $tag
+    cd ../openjdk-git
+    git fetch --all
+    git checkout $tag
```

Comment:
We're happy that we've left this function with git pointing at the tag (do we need to reset later?)

Reply:
the workspace gets wiped out every time